pepper-apis/src/main/java/org/broadinstitute/ddp/util/StudyDataLoaderMain.java

```diff
@@ -94,7 +95,7 @@ public static void main(String[] args) throws Exception {
         options.addOption("m", false, "Also load mailing address");
         options.addOption("o", true, "Output/Report csv file");
         options.addOption("u", true, "Users to migrate (comma separated list of altpids)");
-        options.addOption("c", true, "Auth0 client name");
+        options.addOption("c", true, "Auth0 client id");
```

Comment:
CC @ssettipalli since he's the data  loader expert ;)

Reply:
Debated to user client_id or client name and ended up using client name because Its more human readable and consistent across regions for a study. 
Looks like I need to use tenantId as well in addition to client Id , not a problem.
StudyDataLoader changes look good.  :+1:

Reply:
Thank you. The problem here is that if eradicate `clientName` completely as @zyme suggests above, we won't be able to use it anymore and `auth0ClientId` will remain our only option.

Reply:
👍